test/files/pos/xml-nodebuffer.scala

```diff
@@ -0,0 +1,11 @@
+// scalac: -Ystop-after:parser
+//
+
+import scala.xml.NodeBuffer
+
+object foo {
+  // SI-9027
+  // xml-nodebuffer.scala:10: error: ';' expected but '.' found.
+  val nodeBuffer: NodeBuffer = <hello/><world/>
+  nodeBuffer.map(println)
```

Comment:
Should be `foreach`, so we can look back without any regrets. I mean so we can look at ourselves in the mirror.